AUTHORS.rst

```diff
@@ -1,3 +1,6 @@
+Authors
+=======
+
```

Comment:
I think we need to add the current statement about Billinge group authors here from cookie cutter (but don't delete other stuff.)